index.html

```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
                </p>
             </div>
 
-            <div class="contact-form">
-               <input type="text" placeholder="Text Input" class="user-text">
-               <input type="text" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email">
-               <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here"></textarea>
-               <button class="see-projects project-btn">
-                  See Project
-               </button>
-            </div>
+            <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
+               <ul>
+                  <li>
+                     <label for="user-name"></label>
+                     <input type="text" placeholder="Enter your name" class="user-name" id="user-name" name="user-name" maxlength="30" required >
+                  </li>
+                  <li>
+                     <label for="email"></label>
+                     <input type="email" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email" required >
+
+                  </li>
+                  <li>
+                     <label for="user-text"></label>
+                     <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here" maxlength="500" required></textarea>
+
+                  </li>
+                  <li>
+                     <button class="see-projects project-btn" type="submit" value="send">
+                        See Project
+                     </button>
+
+                  </li>
+               </ul>
+             
+            </form>
```

Comment:
- [ ] Please know that as the previous reviewer mentioned, your website should be responsive for almost all mobile phones, which means the minimum width should be `375px` but that doesn't mean it's the only width, your design should be responsive to up to the width `768px`, in your case to solve this you should remove the `fixed width` you put in your CSS, and use flexbox to align the content in the webpage accordingly this way any screens between the widths I mentioned would display your website correctly. 😁 Also, it's worth knowing that later you'll have to adapt your webpage to be responsive for desktop as well (you don't have to worry about that now) this means if you set up the correct foundation now by using the correct methods, it'll be easy for you to adapt your website to any screen size. 😉

- [ ] Also you forgot to update the placeholder for the email input, as it should be something like `Email`, kindly change that. 😁

Reply:
Thank you so much  @mirouhml for this detailed review